packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -359,6 +360,14 @@ class Draggable<T extends Object> extends StatefulWidget {
   /// Defaults to [HitTestBehavior.deferToChild].
   final HitTestBehavior hitTestBehavior;
 
+  /// Called when interaction starts. This limits the dragging behavior
+  /// for custom clicks (such as scroll click).
+  /// When value is [], allow no interactions.
+  /// When value is null, allow all interactions.
+  ///
+  /// Defaults to [kPrimaryButton].
+  final List<int>? allowListPointer;
```

Comment:
If I'm not mistaken, when the previous value defaulted,
```dart
this.allowListPointer = const <int>[kPrimaryButton]
```
this value cannot be null.